release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2396,6 +2396,116 @@ $ ./openshift-install wait-for install-complete
+
+* If secure boot is currently disabled and you try to enable it using ZTP, the cluster installation does not start. When secure boot is enabled through ZTP, the boot options are configured before the virtual CD is attached. Therefore, the first boot from the existing hard disk has the secure boot turned on. The cluster installation gets stuck because the system never boots from the CD. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])
+
+* Using {rh-rhacm-first}, spoke cluster deployments on Dell PowerEdge R640 servers are blocked when the virtual media does not disconnect the ISO in the iDRAC console after writing the image to the disk. As a workaround, disconnect the ISO manually through the Virtual Media tab in the iDRAC console. 
+(link:https://issues.redhat.com/browse/OCPBUGSM-45884[*OCPBUGSM-45884*])
+
+* Low-latency applications that rely on high-resolution timers to wake up their threads may very infrequently experience higher wake up latencies than expected. 
+Although the expected wake up latency is under 20us, latencies exceeding this can occasionally be seen when running the cyclictest tool for long durations (24 hours or more). 
+Testing has shown that wake up latencies are under 20us for over 99.999999% of the samples.
+(link:https://issues.redhat.com/browse/RHELPLAN-138733[*RHELPLAN-138733*])
+
+* A Chapman Beach NIC from Intel must be installed in a bifurcated PCIe slot to ensure that both ports are visible.  A limitation also exists in the current devlink tooling in RHEL 8.6 which prevents the configuration of 2 ports in the bifurcated PCIe slot.
+(link:https://issues.redhat.com/browse/RHELPLAN-142458[*RHELPLAN-142458*])
+
+* Disabling an SR-IOV VF when a port goes down can cause a 3-4 second delay with Intel NICs. 
+(link:https://issues.redhat.com/browse/RHELPLAN-126931[*RHELPLAN-126931*])
+
+* When using Intel NICs, IPV6 traffic stops when an SR-IOV VF is assigned an IPV6 address. 
+   (link:https://issues.redhat.com/browse/RHELPLAN-137741[RHELPLAN-137741])
+
+* When using VLAN strip offloading, the offload flag (`ol_flag`) is not consistently set correctly with the iavf driver.
+(link:https://issues.redhat.com/browse/RHELPLAN-141240[*RHELPLAN-141240*])
+ 
+* A deadlock can occur if an allocation fails during a configuration change with the ice driver.
+(link:https://issues.redhat.com/browse/RHELPLAN-130855[*RHELPLAN-130855*])
+
+* SR-IOV VFs send GARP packets with the wrong MAC address when using Intel NICs.
+(link:https://issues.redhat.com/browse/RHELPLAN-140971[*RHELPLAN-140971*])
+
+* When using the GitOps ZTP method of managing clusters and deleting a cluster which has not completed installation, the cleanup of the cluster namespace on the hub cluster may hang indefinitely. The workaround to allow the namespace deletion to complete is as follows:
+. Remove the `baremetalhost.metal3.io` finalizer from two CRs in the cluster namespace. 
```

Comment:
These are not 3 steps, the first sentence is a clarification for the 2 steps that follow.

Reply:
> When using the GitOps ZTP method of managing clusters and deleting a cluster which has not completed installation, the cleanup of the cluster namespace on the hub cluster may hang indefinitely. **To complete the namespace deletion, remove the [baremetalhost.metal3.io](http://baremetalhost.metal3.io/) finalizer from two CRs in the cluster namespace:**
> . Remove the `baremetalhost.metal3.io` finalizer [...]
WDYT?